common/include/pcl/impl/point_types.hpp

```diff
@@ -822,22 +828,30 @@ namespace pcl
     PCL_MAKE_ALIGNED_OPERATOR_NEW
   };
 
-
-
```

Comment:
Please always make sure that the diff looks as clean as possible, and only change lines that are directly related to the topic of the commit. So e.g. don't add or remove random lines or change whitespace (unless the commit is specifically made to improve formatting, but that is not the case here)

Reply:
Okay so I'll add the two space lines